src/components/LandingPage/LandingPage.js

```diff
@@ -28,11 +28,9 @@ export const IntroHeading = styled(H1)`
   margin-right: ${props => props.theme.contentPaddingSmall}px;
 
   @media (min-width: ${baselineBreakpoint}px) {
-    margin-top: ${14 * baselineLarge}px;
-    margin-left: ${props =>
-      props.theme.contentPaddingLarge + grid.sideMargin}px;
-    margin-right: ${props =>
-      props.theme.contentPaddingLarge + grid.sideMargin}px;
+    margin-top: ${10 * baselineLarge}px;
+    margin-left: ${5 * baselineLarge + grid.sideMargin}px;
+    margin-right: ${5 * baselineLarge + grid.sideMargin}px;
```

Comment:
Same here, no baseline vars for horizontal spacing. That's why we have the content padding vars in the theme.